spring-rabbit/src/test/java/org/springframework/amqp/rabbit/core/RabbitTemplateIntegrationTests.java

```diff
@@ -1355,9 +1366,15 @@ public Message handleMessage(Message message) {
 			Object result = this.template.convertSendAndReceive("foo");
 			container.stop();
 			assertEquals("FOO", result);
-			assertThat(replyToWas.get(), startsWith(Address.AMQ_RABBITMQ_REPLY_TO));
+			if (tempQueue) {
```

Comment:
Assertion could be moved to testSendAndReceiveNeverFastt test method and we could have a 2 tests one which test with tempQueue = false and one with tempQueue = true.